datastore/store.go

```diff
@@ -47,13 +47,13 @@ type Config struct {
 	// ATXSize must be larger than the sum of all ATXs in last 2 epochs to be effective
 	// (Epoch 12: ~ 300k, Epoch 11: ~ 200k)
 	ATXSize         int `mapstructure:"atx-size"`
-	MalfeasenceSize int `mapstructure:"malfeasence-size"`
+	MalfeasanceSize int `mapstructure:"malfeasance-size"`
```

Comment:
This might break existing configurations, but I don't think someone would intentionally change the cache size for malfeasance proofs, so 🤷